BUILDING_MULTI_ARCH.md

```diff
@@ -0,0 +1,31 @@
+# BUILDING MULTI-ARCHITECTURE IMAGES
+
+The Docker images for OpenKruise can be built by running:
```

Comment:
We can explain `make docker-build` for building test images and `make docker-buildx` for building multi-arch stable images.

Reply:
Done.